MSBuild.lutconfig

```diff
@@ -0,0 +1,7 @@
+<LUTConfig Version="1.0">
+  <Repository />
+  <ParallelBuilds>true</ParallelBuilds>
+  <ParallelTestRuns>false</ParallelTestRuns>
```

Comment:
Also curious about this - Is this because tests from different test projects modify the same files on disk?

(Note that this setting will execute tests from different test projects in parallel - but tests within the same test project will still execute sequentially.)

Reply:
This one was defensive and I didn't actually do due diligence on it. Long long ago our tests weren't able to run in parallel (we turn that off in the xunit runner, too), and I saw the option and just clicked it immediately.

Reply:
Running in parallel may give you some speedup in terms of feedback (from LUT glyphs) as you make changes (especially changes that impact a lot of test assemblies). However, this also means test execution would consume significant amount of CPU on your machine, so it is a trade-off...

I'd recommend defaulting to ParallelTestRuns=true if your tests are safe to run in parallel. I am not sure if the xUnit runner setting applies to parallelism for tests within the same test assembly versus across test assemblies. If it is the former, then I think you could safely set ParallelTestRuns=true since LUT's setting only controls parallelism across test assemblies...

Reply:
I've been running with parallel locally and it seems fine so I'll turn it on here. Thanks for the explanation!